### PR TITLE
Fixed broken hyperlinks in carbon-graphs README

### LINKED ISSUE #198: Broken hyperlinks on the carbon-docs package
# Story

## Description

As a developer, I would like to fix the hyperlinks in the [carbon-graphs README](https://github.com/cerner/terra-graphs/blob/f45e1fbb01025f1a502d2714d82c6e7ad94bf30e/packages/carbon-graphs/README.md).

## Motivation

So that the documentation can be accessed by consumers. 

## Definition of Done

We'll know we're done when the hyperlinks are working again.

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Closes #198 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/carbon-graphs/README.md [hunk 1]
```diff
@@ -86,25 +86,25 @@ The data can now by plotted by as a line graph by using the following:
...
--   [Paired Result](../terra-docs/docs/controls/PairedResult.md#usage)
--   [Bar](../terra-docs/docs/controls/Bar.md#usage)
--   [Timeline](../terra-docs/docs/controls/Timeline.md#usage)
--   [Pie](../terra-docs/docs/controls/Pie.md#usage)
--   [Gantt](../terra-docs/docs/controls/Gantt.md#usage)
--   [Scatter](../terra-docs/docs/controls/Scatter.md#usage)
--   [Bubble](../terra-docs/docs/controls/Bubble.md#usage)
-    -   [Bubble Single Dataset](../terra-docs/docs/controls/BubbleSingleDataset.md)
-    -   [Bubble Multiple Dataset](../terra-docs/docs/controls/BubbleMultipleDataset.md)
+-   [Line](../terra-graphs-docs/docs/controls/Line.md#usage)
```
Comment (AshishMotanamGurunadham) [inline packages/carbon-graphs/README.md:96]: Looks like these links are not working as expected. 
image

Comment (sdadn) [reply packages/carbon-graphs/README.md:96]: They should be working on GitHub. If you select `Rich Text Difference` and try the links, they should work.

CleanShot 2021-08-04 at 20 18 08@2x

Comment (AshishMotanamGurunadham) [reply packages/carbon-graphs/README.md]: Even in local, it dosen't look good
image

**Review by tbiethman — APPROVED** [review]

File: packages/carbon-graphs/README.md (hunk 1 above)
Comment (sdadn) [reply packages/carbon-graphs/README.md]: This is also happening in other .md files that weren't updated so I think it's a rendering issue. Either way, it's beyond the scope of this story.

**Review by benbcai — APPROVED** [review]

**Review by AshishMotanamGurunadham — APPROVED** [review]

**Pull request merged**